<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Goodbye Nahimic</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="GoButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>431</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>GO</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="ConfigBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>431</width>
      <height>107</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Config</string>
    </property>
    <widget class="QComboBox" name="MoboManufacturercomboBox">
     <property name="geometry">
      <rect>
       <x>308</x>
       <y>30</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <item>
      <property name="text">
       <string>--Choose One--</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Asus</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>MSI</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="MoboManufactureLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Motherboard manufacturer</string>
     </property>
    </widget>
    <widget class="QLabel" name="BlacklistIDsLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>341</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Blacklist driver IDs (may help prevent it being reinstalled)</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="BlacklistIDsCheckbox">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="LogBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>431</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="title">
     <string>Log</string>
    </property>
    <widget class="QTextEdit" name="Log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>411</width>
       <height>251</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:400;&quot;&gt;[LOG] test :)&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-weight:400;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="textInteractionFlags">
      <set>Qt::NoTextInteraction</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>450</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMade_by_the_fern">
    <property name="title">
     <string>Made by the fern :)</string>
    </property>
    <addaction name="actionCheck_out_my_website"/>
   </widget>
   <addaction name="menuMade_by_the_fern"/>
  </widget>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionCheck_out_my_website">
   <property name="text">
    <string>Check out my website!</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>MoboManufacturercomboBox</tabstop>
  <tabstop>BlacklistIDsCheckbox</tabstop>
  <tabstop>GoButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
